src/Modules/Backend/Domain/ModuleSettings/ModuleSettingsType.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace ForkCMS\Modules\Backend\Domain\ModuleSettings;
+
+use ForkCMS\Core\Domain\Form\SwitchType;
+use ForkCMS\Modules\Extensions\Domain\Module\Command\ChangeModuleSettings;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\Expression;
+
+final class ModuleSettingsType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
```

Comment:
I would add a form validator here as well, when 2fa is enabled the other settings should be filled in as well

Reply:
fixed 58e75209c75b9af7a127e37ba4cd6b8debee172f